File: AmericanExpress/src/main/java/com/braintreepayments/api/AmericanExpressAnalytics.java
package com.braintreepayments.api;

enum AmericanExpressAnalytics {

    REWARDS_BALANCE_STARTED("amex:rewards-balance:started"),
    REWARDS_BALANCE_FAILED("amex:rewards-balance:failed"),
    REWARDS_BALANCE_SUCCEEDED("amex:rewards-balance:succeeded");

    private final String event;

    AmericanExpressAnalytics(String event) {
        this.event = event;
    }

    String getEvent() {
        return event;
    }
}

File: AmericanExpress/src/main/java/com/braintreepayments/api/AmericanExpressAnalytics.kt
package com.braintreepayments.api

internal enum class AmericanExpressAnalytics(@JvmField val event: String) {

    REWARDS_BALANCE_STARTED("amex:rewards-balance:started"),
    REWARDS_BALANCE_FAILED("amex:rewards-balance:failed"),
    REWARDS_BALANCE_SUCCEEDED("amex:rewards-balance:succeeded")
}
